monitor-metadata-policies/zone-defaults/dfw.json

```diff
@@ -0,0 +1,8 @@
+{
+  "scope": "GLOBAL",
+  "subscope": null,
+  "targetClassName": "RemotePlugin",
+  "valueType": "STRING_LIST",
+  "key": "monitoring_zones_default",
```

Comment:
should this be dfw?

Reply:
Nope.  This policy relates to resources that don't have a `region`.  If a monitor with no zones specified is bound to those we treat it as the `default` region - https://github.com/racker/salus-telemetry-model/blob/master/src/main/java/com/rackspace/salus/telemetry/entities/MetadataPolicy.java#L38

Reply:
Scratch that.  You're right.  My comment before was when viewing from the "Conversation" page and I missed the file this is in.